UseCases/SupplyChain/README.md

```diff
@@ -0,0 +1,41 @@
+# Broader Supply Chain Value Propositions and Use Cases
+To fully appreciate the value of the OSIM use cases 
+requires understanding of the broader supply chain uses
+in which OSIM plays a role.
+For example to see the value of information modeling to 
+software bill of materials (SBOM), 
+you first have to undershand the value of SBOMs.
+
+The use cases in this directory are broader than OSIM 
+and show the value of supply chain transparence in general.
+
+The purpose of these use cases is to demonstrate the benefits of 
+Software Bills of Materials (SBOMs) to software Producers and Consumers. 
+It strives to answer the questions: 
+“Once I generate or receive an SBOM, what do I do with it?” and 
```

Comment:
nit: suggest to use plain ASCII double quote characters.

Reply:
I don't understand this comment as they are plain ascii double quote characters on both the raw and displayed text (at least on my browser).

Reply:
It is only a nit, but when I copy the content of that element ^^^ I get:
> “Once I generate or receive an SBOM, what do I do with it?” and 

These are typographical doublequotes &ldquo; amd &rdquo; (or &#8220; and &#8221;).